src/main/java/com/powsybl/networkconverterserver/NetworkConverterSwaggerConfig.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.networkconverterserver;
```

Comment:
General remark => com.powsybl.network.conversion.server (I known we used com.powsybl.caseserver but is was a mistake as com.powsybl.case could make sense for another project it should have been com.powsybl.case.server, to fix later)